snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -19,17 +19,24 @@ import XCTest
 var deviceLanguage = ""
 var locale = ""
 
-@available(*, deprecated, message: "use setupSnapshot: instead")
-func setLanguage(_ app: XCUIApplication) {
-    setupSnapshot(app)
-}
-
 func setupSnapshot(_ app: XCUIApplication) {
     Snapshot.setupSnapshot(app)
 }
 
-func snapshot(_ name: String, waitForLoadingIndicator: Bool = true) {
-    Snapshot.snapshot(name, waitForLoadingIndicator: waitForLoadingIndicator)
+@available(*, deprecated, renamed: "snapshot(_:timeWaitingForIdle:)")
+func snapshot(_ name: String, waitForLoadingIndicator: Bool) {
```

Comment:
Even if we're deprecating this method,`waitForLoadingIndicator` should have the same default value to avoid changing behavior in existing code, right?

Also, I'm not 100% sure we should deprecate this. Seems like we could just keep both methods as two different interfaces.

Reply:
If there are two methods with `_ name: String` as first parameter and a (different) second parameter which both have a default value, calling `snapshot("test")` will result in an error stating "Ambiguous use of 'snapshot'". That's why I removed this default value.

The behavior for people who just call `snapshot("test")` will not change, it will still wait. But only the default value of 20 seconds instead of indefinitely.

Yes we could keep it. It just seemed cleaner to me to have just one interface. Shall I remove the deprecation?

Reply:
Ah makes sense. I would leave it the way it is. Thanks for the explanation.